README.md

```diff
@@ -103,7 +103,28 @@ You can use a `.env` file to supply environment config locally. Create a fresh o
 | EVIDENCE_API_TOKEN_DOCUMENT_TYPES_GET     |                                                                     |                                     |
 | EVIDENCE_API_TOKEN_EVIDENCE_REQUESTS_GET  |                                                                     |                                     |
 | EVIDENCE_API_TOKEN_EVIDENCE_REQUESTS_POST |                                                                     |                                     |
+| EVIDENCE_API_TOKEN_RESIDENTS_GET          |                                                                     |                                     |
```

Comment:
are all these tokens definitely going to be different?

Reply:
I believe so yes - each HTTP verb for a given path has unique tokens created